<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>definedFilter</class>
 <widget class="QDialog" name="definedFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit0">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>19</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入一个3*3的滤波器模板：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>definedFilter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>definedFilter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
